### ISSUE TITLE
Support interpolation for IN (...) queries

### ISSUE THREAD

**Issue by gaeljw** [zio/zio-jdbc#104]
Unless I missed something, it's not possible to use SQL string interpolation with `WHERE field IN (values)` today.

For instance the following does not work:

```scala
val list = Seq("a","b")

sql"""SELECT field
          FROM table
          WHERE other_field IN ($list)"""
```

This raises conversion error in the JDBC driver used (H2 in my case).

And I couldn't find any workaround.

**Comment by gaeljw** [comment]
Adding that this is actually weird that it even works as strings are not quoted in my `mkstring` version :thinking:

**Comment by gaeljw** [comment]
Well, one workaround is to completely bypass string interpolation with `stringToSql("""...""")` and building manually the string query.

**Comment by gaeljw** [comment]
Solved in v0.1.0.

**Comment by gaeljw** [comment]
It seems multiple clauses are not properly supported though: https://github.com/zio/zio-jdbc/issues/152
